converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/Worker/DicomCastWorker.cs

```diff
@@ -49,25 +54,29 @@ public class DicomCastWorker : IDicomCastWorker
     private readonly ILogger _logger;
     private readonly IHostApplicationLifetime _hostApplicationLifetime;
     private readonly IFhirService _fhirService;
+    private readonly TelemetryClient _telemetryClient;
```

Comment:
Where is this telemetry client registered?

Reply:
Probably it's in the PaaS side, but I just want to confirm.

Reply:
We  use AddApplicationInsightsTelemetry in both paas and startup class here to register this clinet.